packages/validators/package.json

```diff
@@ -45,5 +45,9 @@
     "repository": {
         "type": "git",
         "url": "https://github.com/iotaledger/iota.js.git/tree/develop/packages/crypto"
+    },
+    "dependencies": {
+        "@iota/converter": "^1.0.0-alpha.1",
+        "@iota/transaction-converter": "^1.0.0-alpha.1"
```

Comment:
I think this introduces a circular dependancy problem; `transaction-converter` requires `validators` and vice-versa. We can solve this by changing `isTransactionHash()` to accept transaction trytes instead of transaction object (see comment bellow), and extracting `transactionHash()` from `transaction-converter`.